### ISSUE TITLE
Translate all UI text to English

### ISSUE THREAD

**Issue by Ilyas-SEKHSOUKHI** [Ilyas-SEKHSOUKHI/BROWSERMATE#1]
Currently, all phrases in the BrowsMate project are not in English.
To improve accessibility and reach a wider audience, all user-facing text should be translated into English.
